rust/cardano-chain-follower/examples/follow_chains.rs

```diff
@@ -200,7 +190,7 @@ async fn follow_for(network: Network, matches: ArgMatches) {
         // If these become true, we will show all blocks from the follower.
         follow_all = follow_all
             || (!chain_update.immutable() && all_live_blocks)
-            || ((chain_update.data.fork() > 1) && all_tip_blocks);
+            || ((chain_update.data.fork() > 1.into()) && all_tip_blocks);
```

Comment:
Suggestion, the `Fork` type probably needs some constants and a method or two for this.
1. `IMMUTABLE_FORK` = 0
2. `BACKFILL_FORK` = 1

`is_live()` returns `true` if chain_update.data.fork() > 1.  So then this code could become:

```
 || (chain_update.data.fork().is_live() && all_tip_blocks);
```
which is clearerer.

The `Fork` type could probably use `is_immutable()` and `is_backfill()` to identify which kind of fork it is, as well.  Although I can't think of an immediate use for those, they would be good to have on the type.